README.md

```diff
@@ -11,21 +11,40 @@ It is pretty straightforward and if you are familiar with the Express.js framewo
 
 ## Installation 🚀
 
-0. Make sure you have the gnu g++ compiler installed on your system. You can install it by running the following command:
+0. Make sure you have the a c++ compiler (such as [g++](https://gcc.gnu.org/) or [clang](https://clang.llvm.org/)) installed on your system.
```

Comment:
No need to include links to compilers if you are mentioning how to install them, but okay..